app/src/test/java/org/oppia/android/app/testing/PlatformParameterIntegrationTest.kt

```diff
@@ -94,6 +126,12 @@ class PlatformParameterIntegrationTest {
   fun setUp() {
     setUpTestApplicationComponent()
     testCoroutineDispatchers.registerIdlingResource()
+    context = InstrumentationRegistry.getInstrumentation().context
```

Comment:
Why do we use this instead of @Inject for the context?

Reply:
Changed it to @Inject format. I was debugging with InstrumentationRegistry I guess.